samples/shields/x_nucleo_cca02m1/README.rst

```diff
@@ -0,0 +1,139 @@
+.. _x-nucleo-cca02m1-sample:
+
+X-NUCLEO-CCA02M1: Digital MEMS microphones SHIELD for STM32 Nucleo
+##################################################################
+
+Overview
+********
+This sample enables the two digital MEMS microphones on X-NUCLEO-CCA02M1
+shields
+
+This sample provides an example of how to acquire audio through
+the two digital MEMS microphones on X-NUCLEO-CCA02M1 sheild.
+The microphone generates a PDM stream which is acquired through I2S.
+The PDM stream is then converted to PCM using the OpenPDMFilter external
+library.
+
+Requirements
+************
+
+This sample communicates over I2C with the X-NUCLEO-CCA02M1 shield
+stacked on a board with an Arduino connector. The board's I2C must be
+configured for the I2C Arduino connector (both for pin muxing
+and device tree). See for example the :ref:`nucleo_f411re_board` board
+source code:
+
+- :zephyr_file:`boards/arm/nucleo_f411re/nucleo_f411re.dts`
+- :zephyr_file:`boards/arm/nucleo_f411re/pinmux.c`
+
+References
+**********
+
+- X-NUCLEO-CCA02M1: https://www.st.com/en/ecosystems/x-nucleo-cca02m1.html
+
+Building and Running
+********************
+
+This sample runs with X-NUCLEO-CCA02M1 stacked on any board with a matching
+Arduino connector. For this example, we use a :ref:`nucleo_f411re_board` board.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/shields/x_nucleo_cca02m1
+   :board: nucleo_f411re
+   :goals: build
+   :compact:
+
+Sample Output
+=============
+
+The example acquires 1s of audio and prints out the PCM stream on COM port.
+The acquisition starts immediately after the reset button is pressed.
+
+The characteristics of the PCM audio are hardcoded in the example:
+
+- 16KHz sample rate
+- 16 bits per sample
+- 2 channel (stereo)
+
+One second of acquisition at a 2 channels 16KHz sampling rate yields 32,000 16-bit samples.
+The microphone PDM requested clock should lead the MP34DT05 driver to select an
+oversampling/decimation factor to result in approximately a 2MHz bit clock.
+
+See pcm and pdm configuration in file :zephyr_file:`samples/shields/x_nucleo_cca02m1/src/main.c`.
+
+.. note:: It is possible to change the AUDIO_FREQ to 32000 acquiring only 500 ms.
+
+At the end of the acquisition the PCM data will be printed on the terminal
+emulator in either binary or ASCII format. The output is controlled by
+following macro, off by default, in :zephyr_file:`samples/shields/x_nucleo_cca02m1/src/main.c`:
+
+* :c:macro:`PCM_OUTPUT_IN_ASCII`
+
+Binary PCM Output
+-----------------
+
+The nucleo F411RE board presents itself as USB CDC class to host, which will so use the
```

Comment:
Let's change this to:
```
The Nucleo F411RE board presents itself to the host
as a USB CDC class, and will use ``/dev/ttyACM0``
device for communication. The ``/dev/ttyACM0`` port
must be configured in raw mode to avoid having
special characters (such as :kbd:`CNTL-Z` or kbd:`CNTL-D`)
processed or 'cooked' out.
```

Reply:
it sounds better, ok.